### PR TITLE
IR-5-Getting redflag Incidents. 

### PR DESCRIPTION
**Pull request by katusiimeconrad**
Implemented the method getRedflagIncidents().

The method can be tested in the Views/iReported/main.
It relies on SavingIncident (IR-2-Saving-an-Incident) and Exceptions (IR-2-Implementing-Exceptions)

`System.out.println(incidentHelper.getRedflagIncidents());`

### CODE REVIEW

**Review by MutebiAlexKato — APPROVED** [review]

**Review by ttcollins — APPROVED** [review]
Great work sir

**Review by iandrew8 — APPROVED** [review]

**Review by benon123 — CHANGES_REQUESTED** [review]
delete all cached files 
using git rm -r --cached
add metadata to gitignore

**Review by Golder12 — APPROVED** [review]

**Review by NakagweSharifah — APPROVED** [review]

**Review by benon123 — APPROVED** [review]

**Review by iandrew8 — APPROVED** [review]

**Review by reifred — CHANGES_REQUESTED** [review]
Kindly ignore all unwanted files in the .gitignore file

**Comment by katusiimeconrad** [comment]
> Kindly ignore all unwanted files in the .gitignore file

Has been resolved . Add the .idea files to .gitignore

File: src/org/pahappa/systems/models/Type.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package org.pahappa.systems.models;
+
+public enum Type {
+    REDFLAG,
+    INTERVENTION;
+}
```
Comment (reifred) [inline src/org/pahappa/systems/models/Type.java]: We already have a Type enum in the enums package. I don't know why you decided to create another one.

**Review by reifred — CHANGES_REQUESTED** [review]
Kindly ignore all unwanted files. You have a lot of changed files
Screen Shot 2021-09-10 at 9 49 43 PM

File: src/org/pahappa/systems/models/Type.java (hunk 1 above)
Comment (katusiimeconrad) [reply src/org/pahappa/systems/models/Type.java]: Will Delete the File. 
I had created it prior for testing before merging with Benon's work

Comment (katusiimeconrad) [reply src/org/pahappa/systems/models/Type.java]: Deleted File, Updated IncidentServiceImpl.java and iReporter.java to reflect the new changes.

**Review by reifred — CHANGES_REQUESTED** [review]
@ttcollins am waiting you to remove unnecessary files in your PR.
Screen Shot 2021-09-12 at 12 12 15 PM

**Comment by katusiimeconrad** [comment]
@reifred All unnecessary files were deleted, and the Merge conflicts resolved. 
All set to Merge

**Review by reifred — APPROVED** [review]

**Pull request merged**